.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  specs:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby-version: ['3.0', '2.7', '2.6', '2.5']
```

Comment:
@rahim We do pay by the minute for Actions usage. I realize Instant API has low churn, but also that Ruby 2.5 reach EOL in March 2021, so _should_ we attempt to support it?

Reply:
Without checking I've been hoping the actions usage here will fall under the free open source allowances rather than under the org.

When we add `3.1` we could drop `2.5` – the reality here is that if we'd had this in place since the beginning we'd have averaged only a few minutes per year of usage in total - change here is rare.

Reply:
great point given that this is a public repo 👍